index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='https://unpkg.com/boxicons@2.0.7/css/boxicons.min.css' rel='stylesheet'>
+    <link rel="stylesheet" href="style.css">
+    <title>Conference-capstone</title>
+</head>
+<body>
+    <header class="header">
+        <nav class="nav">
+            <div class="icon" id="hamburger">
+                <i class='bx bx-menu hamburger'></i>
+            </div>
+            <div class="logo">
+                <a href="index.html">Logo</a>
+            </div>
+            <ul class="nav-list">
+                <li class="nav-item">
+                    <div class="icon">
+                        <i class='bx bx-x close' id="close-icon"></i>
+                    </div>
+                </li>
+                <li class="nav-item"><a href="index.html" class="nav-link">Home</a></li>
+                <li class="nav-item"><a href="about.html" class="nav-link">About</a></li>
+            </ul>
+        </nav>
+        <div class="hero-container">
+            <div class="hero-content">
+                <p class="hero-content__text">"Hello! Sharing world"</p>
+                <h1 class="hero-content__heading">
+                    CREATIVE COMMONS GLOBAL SUMMIT 2021
+                </h1>
+                <p class="hero-content__desc">
+                    A joyful celebration bellying in the value of openness and
+                    sharing, creating positive change with people from over 80 
+                    countries is taking place in October, in Nigeria
+                </p>
+                <h2 class="hero-content__date">
+                    2021.10.15(THU) ~ 16(FRI)
+                </h2>
+                <span class="hero-content__place">@National Theater of Nigeria</span>
+            </div>
+        </div>
+    </header>
+    <main>
+        <section>
+            <div class="container bg-dark">
+                <div class="program">
```

Comment:
Apply the proper mobile design in the below section as shown below the difference 

#### Original Project
![right3](https://user-images.githubusercontent.com/58376020/123693413-4ddd4300-d858-11eb-9424-c5a629a92554.png)

#### Your Project
![wrong1](https://user-images.githubusercontent.com/58376020/123693445-5afa3200-d858-11eb-8fff-df0329373fb0.png)

Reply:
This change has been implemented well. 👍🏿